test/junit/scala/collection/IterableViewLikeTest.scala

```diff
@@ -4,6 +4,7 @@ import org.junit.Assert._
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
+import language.postfixOps
```

Comment:
A couple of my PRs have featured that feature, but I don't think any made it in. That warning was so annoying, especially when running test.junit while fixing a really annoying bug (annoying because it's working hard not to get fixed). It's like sitting in traffic and someone cuts in front of you. Not evil in itself, but the root of other bad behaviors.